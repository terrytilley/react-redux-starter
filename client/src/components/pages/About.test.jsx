import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import About from './About';

describe('<About />', () => {
  it('should render component', () => {
    const wrapper = shallow(<About />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
