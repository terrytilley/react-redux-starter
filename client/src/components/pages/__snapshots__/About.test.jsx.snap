// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render component 1`] = `
<div>
  <h1>
    About Page
  </h1>
</div>
`;
